pkg/notes/notes.go

```diff
@@ -472,8 +474,9 @@ func ListCommitsWithNotes(
 		// "release note none" that appear in the commit log.
 		exclusionFilters := []string{
 
-			// 'none' or 'n/a' case insensitive with optional trailing whitespace, wrapped in ``` with/without release-note identifier
-			"(?i)```(release-note\\s*)?(none|n/a)?\\s*```",
+			// 'none','n/a','na' case insensitive with optional trailing whitespace, wrapped in ``` with/without release-note identifier
```

Comment:
The issue with the [linked PR](https://github.com/kubernetes/kubernetes/pull/79933) is that we have something like `release-notes` instead of `release-note` in the PR body, I will fix that in a separate PR.

Reply:
See https://github.com/kubernetes/release/pull/871

Reply:
@justaugustus @saschagrunert - The release notes tool will only pull release notes from the `reelease-notes` block. So other code blocks can be added to the PR without any affect on the release notes extractedd.

The [kubernetes/sig-release#763 (comment)](https://github.com/kubernetes/sig-release/pull/763#discussion_r315064356) was resolved.

Reply:
@justaugustus @saschagrunert - re: capturing documentation. I don't think the tool currently captures the links in the `docs` codeblock 100% accurately against the following guidance.

```
Use one or more of the following templates in the docs block for linking to documentation:
- [KEP]: <url>
- [Usage]: <url>
- [Other doc]: <url>
```